### PR TITLE
Mobile beta entry

### PR DESCRIPTION
**Pull request by Proyoyo**
Added mobile beta entry

### CODE REVIEW

File: _posts/2017-11-17-mobile-beta.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
+---
+layout: post
+title:  Open Beta for OpenDota Mobile Application
+date: 2017-11-17
+summary: Open beta for a new OpenDota mobile application
+---
+As some of you may know, the mobile application for OpenDota has been in development for awhile. Now, we are proud to announce that the application is now in Open Beta!
```
Comment (howardchung) [inline _posts/2017-11-17-mobile-beta.md]: "a while"

File: _posts/2017-11-17-mobile-beta.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+The application is developed in React Native so that both the iOS and Android builds are made from the same codebase.
+
+For those of you who are interested in beta testing the OpenDota mobile application, please send me (via Reddit pm or Discord pm) your email address & device type (e.g. iPhone X, Google Pixel 2, etc).
```
Comment (howardchung) [inline _posts/2017-11-17-mobile-beta.md]: pm should be capitalized

File: _posts/2017-11-17-mobile-beta.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+*We’re planning on releasing this to the app store around end of December - early January.*
+
+**Why does x feature does not exist in the mobile app?**
+
+*Our plan for the app is to reach feature parity with OpenDota website. We will enhance the experience add more features gradually.*
+
+**I’m interested in contributing to the app development. How can I help?**
+
+*You can contact me on [Discord](https://discordapp.com/invite/0o5SQGbXuWCNDcaF) #mobile channel and let me know what you want to implement. Or hit up our [Github issue page](https://github.com/odota/mobile/issues) and submit a PR related to the issue.*
```
Comment (howardchung) [inline _posts/2017-11-17-mobile-beta.md]: mention your Discord name (Proyoyo)

**Comment by howardchung** [comment]
@nicholashh and @albertcui can you review?

**Review by howardchung — APPROVED** [review]

File: _posts/2017-11-17-mobile-beta.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+
+## FAQ
+
+**When will this be released to the app store?**
+
+*We’re planning on releasing this to the app store around end of December - early January.*
+
+**Why does x feature does not exist in the mobile app?**
+
+*Our plan for the app is to reach feature parity with OpenDota website. We will enhance the experience add more features gradually.*
```
Comment (howardchung) [inline _posts/2017-11-17-mobile-beta.md]: "and add'

File: _posts/2017-11-17-mobile-beta.md [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+
+**I’m interested in contributing to the app development. How can I help?**
+
+*You can contact me (Proyoyo) on [Discord](https://discordapp.com/invite/0o5SQGbXuWCNDcaF) #mobile channel and let me know what you want to implement. Or hit up our [Github issue page](https://github.com/odota/mobile/issues) and submit a PR related to the issue.*
```
Comment (howardchung) [inline _posts/2017-11-17-mobile-beta.md]: Github should be GitHub

File: _posts/2017-11-17-mobile-beta.md [hunk 6]
```diff
@@ -0,0 +1,32 @@
+---
+layout: post
+title:  Open Beta for OpenDota Mobile Application
```
Comment (albertcui) [inline _posts/2017-11-17-mobile-beta.md]: A little wordy, "Open Beta for OpenDota Mobile App!"

**Review by albertcui — APPROVED** [review]
Cool, looks good! In general "Application" -> "App" would be nice since it's less wordy.

**Pull request merged**